IdentityCore/src/cache/mac/MSIDMacKeychainTokenCache.m

```diff
@@ -870,6 +1031,119 @@ - (BOOL)clearWithContext:(id<MSIDRequestContext>)context
     return YES;
 }
 
+#pragma mark - Access Control Lists
+
+- (id) accessCreateWithChanceACL:(NSArray<id>*)trustedApplications
```

Comment:
This is a tricky one. This is done because we have a Core Foundation property on self which we are using as a NSObject property which is allowed as long as we do the appropriate bridge cast when required. See below:
https://jeremywsherman.com/blog/2013/01/29/toll-free-__bridging/
Another option is to use the __attribute__ keyword to specify that a Core Foundation property should be treated like an Objective-C object for memory management.